src/mainwindow.cpp

```diff
@@ -1063,6 +1063,7 @@ void MainWindow::loadAppSettings()
     ui->translateButton->setShortcut(settings.translateShortcut());
     ui->swapButton->setShortcut(settings.swapShortcut());
     ui->copyTranslationButton->setShortcut(settings.copyTranslationShortcut());
+    ui->clearButton->setShortcut(settings.clearShortcut());
```

Comment:
This is correct, but you also need to assign other buttons. Search for their names in `mainwindow.ui`

Reply:
I don't think there are buttons for the other actions. I don't have the knowledge to code them either, sadly